galaxy_importer/main.py

```diff
@@ -98,27 +107,43 @@ def call_importer(args, cfg):  # pragma: no cover
 
     Method excluded from pytest unit test coverage, tests exist in tests/integration
     """
-    if not args.file:
-        return collection.import_collection(
-            git_clone_path=os.path.abspath(args.git_clone_path),
-            output_path=os.path.abspath(args.output_path),
-            logger=logger,
-            cfg=cfg,
-        )
-
-    match = FILENAME_REGEXP.match(os.path.basename(args.file))
-    namespace, name, version = match.groups()
-    filename = collection.CollectionFilename(namespace, name, version)
-
-    with open(args.file, "rb") as fh:
+    if args.legacy_role:
```

Comment:
depending on the legacy-role parameter, ansible-lint sometimes gives no output

```
(.env) awc:~$ python -m galaxy_importer.main --legacy-role code/ansible-role-jenkins --namespace awcrosby
Importing with galaxy-importer 0.4.10
Determined role name to be ansible-role-jenkins
Linting role ansible-role-jenkins via ansible-lint...
...ansible-lint run complete
Legacy role loading complete
Importer processing completed successfully
(.env) awc:~$ cd code
(.env) awc:code$ python -m galaxy_importer.main --legacy-role ansible-role-jenkins --namespace awcrosby
Importing with galaxy-importer 0.4.10
Determined role name to be ansible-role-jenkins
Linting role ansible-role-jenkins via ansible-lint...
WARNING: ansible-role-jenkins/molecule/default/java-8.yml:3:3: syntax-check[specific]: 'apt_repository' is not a valid attribute for a Play
...
```

Reply:
will be resolved by strictly allowing directory depth of 1